include/finufft/finufft_core.h

```diff
@@ -182,11 +176,10 @@ template<typename TF> struct FINUFFT_PLAN_T { // the main plan object, fully C++
   std::vector<TF> phiHat2; // " y-axis.
   std::vector<TF> phiHat3; // " z-axis.
 
-  TC *fwBatch = nullptr;   // (batches of) fine grid(s) for the FFT to plan
-                           // & act on. Usually the largest working array.
-                           // Unfortunately this can't be a vector, since
-                           // in the presence of FFTW it must be allocated
-                           // by fftw_malloc.
+  std::vector<TC, xsimd::aligned_allocator<TC, 64>> fwBatch; // (batches of) fine grid(s)
```

Comment:
Amazing!

Reply:
Since 64 is the length of a cache line, and has been since a few decades now, I would say yes. But we can increase this without wasting much memory if we want.

Reply:
I think is safe to leave it as 64 for now. If a new architecture with a 128 cache line comes around we can increase it. We could put a comment in the code that this assumes cache lines size = 64.